app/src/test/java/edu/uc/kansakpb/ikeahjalpare3/FurnitureDataUnitTest.kt

```diff
@@ -45,12 +47,15 @@ class FurnitureDataUnitTest {
     }
 
     private fun createMockData() {
-        var allFurnituresLiveData = MutableLiveData<ArrayList<Furniture>>()
-        var allFurnitures = ArrayList<Furniture>()
+        val allFurnituresLiveData = MutableLiveData<ArrayList<Furniture>>()
+        val allFurnitures = ArrayList<Furniture>()
         //create and add plants to our collection.
-        var KopangDrawerDresser = Furniture("KopangDrawerDresser", "https://www.ikea.com/us/en/assembly_instructions/koppang-drawer-dresser__AA-2066877-1_pub.pdf", "https://www.youtube.com/watch?v=biL6K6OESe4")
-        allFurnitures.add(KopangDrawerDresser)
-        var KopangLamp = Furniture("KopangLamp", "https://www.ikea.com/us/en/assembly_instructions/koppang-drawer-dresser__AA-2066877-1_pub.pdf", "https://www.youtube.com/watch?v=biL6K6OESe4")
+        val kopangDrawerDresser = Furniture("KopangDrawerDresser",
+            "https://www.ikea.com/us/en/assembly_instructions/koppang-drawer-dresser__AA-2066877-1_pub.pdf",
+            "https://www.youtube.com/watch?v=biL6K6OESe4", furnitureName ="KopangDrawerDresser", instructionsManuel = "m" )
+        allFurnitures.add(kopangDrawerDresser)
+        val KopangLamp = Furniture("KopangLamp", "https://www.ikea.com/us/en/assembly_instructions/koppang-drawer-dresser__AA-2066877-1_pub.pdf",
+            "https://www.youtube.com/watch?v=biL6K6OESe4", furnitureName ="KopangDrawerDresser", instructionsManuel = "assembly_instructions")
```

Comment:
Pass furnitureName and instructionsManuel as they were missed.

Reply:
Fix the spelling of manual, while you're in here.  :)